packages/cloud2edge/Chart.yaml

```diff
@@ -11,8 +11,8 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 apiVersion: v1
-version: 0.4.1
-appVersion: 0.4.1
+version: 0.4.2
+appVersion: 0.4.2
```

Comment:
@WatcherWhale Could you revert the changes in the cloud2edge Chart.yaml? They are not needed.
(Changes to the Hono version requirement have already been included in PR #483.)